src/account/utils.cairo

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo vX.Y.Z (account/utils.cairo)
+
+const TRANSACTION_VERSION: u256 = 1;
+// 2**128 + TRANSACTION_VERSION
+const QUERY_VERSION: u256 = 0x100000000000000000000000000000001;
```

Comment:
in a previous PR we decided to keep this next to the account implementation to make it easier/obvious to check what version is supported by an account, and to open the door to supporting different versions in different modules if we wanted to.

any reason to move this here now?

Reply:
if we keep it, i'd go with `constants.cairo` instead of `utils.cairo`

Reply:
> in a previous PR we decided to keep this next to the account implementation to make it easier/obvious to check what version is supported by an account, and to open the door to supporting different versions in different modules if we wanted to.
any reason to move this here now?

It was an anticipatory move because the EthAccount PR has it separated for both account components to use. No strong reason otherwise. I'll switch it back

Reply:
yes I noticed afterwards